Libraries/Core/InitializeCore.js

```diff
@@ -86,18 +86,6 @@ ReactNativeVersionCheck.checkVersions();
 // ERROR: Event loop not supported.
 polyfillGlobal('Promise', () => require('Promise'));
 
-// Set up regenerator.
-polyfillGlobal('regeneratorRuntime', () => {
-  // The require just sets up the global, so make sure when we first
-  // invoke it the global does not exist
-  delete global.regeneratorRuntime;
-  /* $FlowFixMe(>=0.54.0 site=react_native_oss) This comment suppresses an
-   * error found when Flow v0.54 was deployed. To see the error delete this
-   * comment and run Flow. */
-  require('regenerator-runtime/runtime');
```

Comment:
Can you also remove `regenerator-runtime` from the dependencies in `package.json`?

Reply:
We still need it for jest tests, I guess I could move it to dev deps